### ISSUE TITLE
Cannot include `.md` as description in openapi's yaml doc

### ISSUE THREAD

**Issue by AchrafAmil** [readmeio/rdme#814]
We used to be able to push/sync openAPI files like this (can't recall which rdme CLI version it was, but last time it worked for me was May 2022):

```yaml
post:
  summary: Create Foo.
  description:
    $ref: "foo-desc.md"
...
```

where `foo-desc.md` is a basic markdown file.

But today I tried with both `8.6.3` and `7.2.0` and I get the following error:

```
The spec you uploaded has validation errors that must be addressed. Here's what we found:

this.schema.description.trim is not a function

https://docs.readme.com/main/reference and include the following link to your API log: 'https://docs.readme.com/main/logs/d9f98408-5e12-4747-af52-491ffc0f84de'.

```

Could we please bring back the support for sharing/reusing .md files in yaml specs? 😇

**Comment by kanadgupta** [comment]
Hi @AchrafAmil! I believe this is due to a bug in the ReadMe API and not due to any changes with the `rdme` client itself. Will do some investigation on this and report back.

**Comment by kanadgupta** [comment]
@AchrafAmil if possible, could you provide a complete example of an OpenAPI file and the corresponding referenced Markdown files that yields this error? In our testing, `rdme` should dereference the Markdown file properly prior to sending it to the ReadMe API, so I'm not sure how you're seeing that error.

I have a fix in https://github.com/readmeio/oas/pull/764 that should address the confusing error message you're seeing, but again, I'm not sure how you're able to see this error in the first place.

**Comment by erunion** [comment]
Following along with that, how are you using `rdme` and which version? In our testing we found that the current `rdme` was correctly dereferencing the Markdown file into an OpenAPI definition, which was then properly sending it up to the ReadMe API -- so yeah, not sure how you're seeing this! 😥

**Comment by kanadgupta** [comment]
Hi @AchrafAmil! We've released a few fixes in both [`rdme@8.6.4-next.1`](https://github.com/readmeio/rdme/releases/tag/8.6.4-next.1) and in the ReadMe API which should address the error message you're seeing. If you're still seeing errors, feel free to open another issue (along with a reproducible example so we can help you out). Thanks!
